pubtools/_quay/command_executor.py

```diff
@@ -56,7 +56,7 @@ def _run_cmd(self, cmd, err_msg=None, tolerate_err=False, stdin=None):
         """Run a bash command."""
         raise NotImplementedError  # pragma: no cover"
 
-    def skopeo_login(self, username=None, password=None):
+    def skopeo_login(self, host, username=None, password=None):
```

Comment:
Maybe it would be a good idea to specify default value to quay.io?

Reply:
hmm, ok, I can put there a default value